### PR TITLE
[bluetooth.generic] Fix build

### PR DESCRIPTION
**Pull request by cpmeister**
Signed-off-by: Connor Petty <mistercpp2000+gitsignoff@gmail.com>

### CODE REVIEW

File: bundles/org.openhab.binding.bluetooth.generic/src/main/java/org/openhab/binding/bluetooth/generic/internal/GenericBluetoothHandler.java [hunk 1]
```diff
@@ -182,6 +182,7 @@ private void updateThingChannels() {
         }
     }
```
Comment (J-N-K) [inline bundles/org.openhab.binding.bluetooth.generic/src/main/java/org/openhab/binding/bluetooth/generic/internal/GenericBluetoothHandler.java:184]: We‘ve run into that in other places. I would suggest `Objects.requireNonNull(...)` in the Return Statement like we have used in the other places.

Comment (cpmeister) [reply bundles/org.openhab.binding.bluetooth.generic/src/main/java/org/openhab/binding/bluetooth/generic/internal/GenericBluetoothHandler.java:185]: done

**Review by J-N-K — APPROVED** [review]

**Comment by cpmeister** [comment]
I'm confused, the jenkins status page says that the generic binding failed again, but the console says that the binding was build successfully. Is something wrong with jenkins?

**Comment by kaikreuzer** [comment]
Not sure how to read this Jenkins report :-/ . Let's merge and I'll clean the PR-build workspace to see if that fixes such issue for future builds.

**Pull request merged**